packages/augur-ui/src/modules/app/components/side-nav/side-nav.styles.less

```diff
@@ -49,6 +49,34 @@
   }
 }
 
+.SideNavMigrateRep {
+  button {
+    .mono-10-bold;
+    text-transform: uppercase;
+  }
+}
+
+.SideNavMigrateTooltipHint {
+  display: inline;
+  position: relative;
+  top: 0.2rem;
+  left: 0.3rem;
+
+  > svg {
+    max-height: @size-16;
+    max-width: @size-16;
+
+    > circle {
+      fill: @color-secondary-action;
+      stroke: @color-secondary-action-outline;
+    }
+
+    > path {
+      fill: @color-primary-text;
+    }
+  }
+}
```

Comment:
I'm also open to discussing this change. These are copies of the TooltipHint in the common folder, but I couldn't figure out how to add an "&" nested parent selector since the webpack strategy garbles the CSS output. I think it's ok to just keep it and optimize for code re-use later to get this out the door, but if anyone has a better way to do this I'm all ears.